lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,34 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/**
+ * @brief Functions to get and set bit fields in a 32/64-bit register.
+ *
+ * These inline functions allow setting and extracting specific bits in
+ * a register (`reg`) according to a given mask (`mask`). The mask
+ * specifies which bits in the register should be updated or extracted.
+ *
+ * - `set_field`: Modifies specific bits in a register by clearing the
```

Comment:
s/- \`set_field\`/set_field*()/
Please indent the following two lines so they align with the start of the word "Modifies".